src/app.html

```diff
@@ -5,9 +5,9 @@
     <meta name="description" content="" />
     <link rel="icon" href="/favicon.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    %svelte.head%
+    %sveltekit.head%
   </head>
   <body>
-    <div id="svelte">%svelte.body%</div>
+    <div id="svelte">%sveltekit.body%</div>
```

Comment:
`id="svelte"` [was removed](https://github.com/sveltejs/kit/pull/3674)
`style="display: contents"` [was added](https://github.com/sveltejs/kit/pull/7652)

```suggestion
    <div style="display: contents">%sveltekit.body%</div>
```